docker-compose.yml

```diff
@@ -15,3 +15,26 @@ services:
         build: ./user-service
         ports:
             - "3001:3001"
+
+    matching-service:
+        build: ./matching-service
+        ports:
+            - "3006:3006"
+        depends_on:
+            - redis
+            - question-service
+        environment:
+            REDIS_HOST: redis
+            REDIS_PORT: 6379
+            QUESTION_SERVICE_URL: http://question-service:3002/
+
+    redis:
+        image: redis
+        container_name: redis-server
+        ports:
+            - "6379:6379"
+        volumes:
+            - redis_data:/data
+
+volumes:
+    redis_data:
```

Comment:
the added docker commands look great, but should the build for the frontend be './frontend' since you changed the name of the file